go/ms/internal/mscrypto/provider.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package config
+package mscrypto
+
+import (
+	"context"
+	"crypto/x509"
+	"math/rand"
+	"time"
+
+	"github.com/scionproto/scion/go/lib/addr"
+	"github.com/scionproto/scion/go/lib/ctrl/cert_mgmt"
+	"github.com/scionproto/scion/go/lib/infra"
+	"github.com/scionproto/scion/go/lib/scrypto/cppki"
+	"github.com/scionproto/scion/go/lib/serrors"
+	"github.com/scionproto/scion/go/lib/snet"
+	"github.com/scionproto/scion/go/pkg/trust"
+)
+
+//MSEngine implements the trust.Provider interface
+type MSEngine struct {
+	Msgr infra.Messenger
+	IA   addr.IA
+}
+
+func (m MSEngine) NotifyTRC(ctx context.Context, trcId cppki.TRCID,
+	o ...trust.Option) error {
+	return nil
+}
+
+func (m MSEngine) GetChains(ctx context.Context, cq trust.ChainQuery,
+	o ...trust.Option) ([][]*x509.Certificate, error) {
```

Comment:
```suggestion
	o ...trust.Option) ([][]*x509.Certificate, error) {
	
```
Add a newline:
https://scion.docs.anapaya.net/en/latest/contribute/go-style.html#function-declaration-over-multiple-lines